src/operator-docs/transformation/mapTo.ts

```diff
@@ -1,6 +1,49 @@
 import { OperatorDoc } from '../operator.model';
 
 export const mapTo: OperatorDoc = {
-  'name': 'mapTo',
-  'operatorType': 'transformation'
+  name: 'mapTo',
+  operatorType: 'transformation',
+  signature: 'public mapTo(value: any): Observable',
+  parameters: [
+    {
+      name: 'value',
+      type: 'any',
+      attribute: '',
+      description: `The value to map each source value to.`
+    }
+  ],
+  useInteractiveMarbles: true,
+  marbleUrl: 'http://reactivex.io/rxjs/img/mapTo.png',
+  shortDescription: {
+    description: `Emits the given constant value on the output Observable every time the source Observable emits a value.`
+  },
+  walkthrough: {
+    description: `<p>
+        Takes a constant
+        <code>value</code> as argument,
+        and emits that whenever the source Observable emits a value.
+        In other words, ignores the actual source value,
+        and simply uses the emission moment to know when to emit the given
+       <code>value</code>.
+       <p>`
+  },
+  examples: [
+    {
+      name: "Map every click to the string 'Hi'",
+      code: `
+      import { fromEvent } from 'rxjs/observable/fromEvent';
+      import { mapTo } from 'rxjs/operators';
+
+      const clicks = Rx.Observable.fromEvent(document, 'click');
+      const greetings = clicks.mapTo('Hi');
```

Comment:
Is it possible to modify the code to use just the 'fromEvent' (without the Rx.Observable. part)? The Rx.Observable part is unnecessary when importing it with ESM.
And in our examples we prefer to use the pipeable operators and its likewise notation
clicks.pipe( mapTo( ... ))
This will make it different than what you show with the jsbin, but for now that's ok.